transforms/dec128.js

```diff
@@ -0,0 +1,78 @@
+const DECIMAL_LITERAL = "DecimalLiteral";
+
+const opToName = {
```

Comment:
Nit not related to Babel: this feels like it should be a map, so that then you can also use `.has` rather than `.includes` on `Object.keys`!